.travis.yml

```diff
@@ -19,8 +19,11 @@ before_script:
   - version="$(awk '$2 == "FREESWITCH_VERSION" { print $3; exit }' Dockerfile | cut -d'~' -f1)"
   - docker pull "$IMAGE"
 
-# Build the container, run it, wait for it to start, then stop it.
+# Build the modules container runs it, modules are copied to build/modules.
+# Build the freeswitch container, run it, wait for it to start, then stop it.
 script:
+  - docker build -f Dockerfile.modules -t fs-modules .
+  - docker run -v $PWD/build/modules:/tmp/build/modules/ -p 6780:6780 fs-modules
```

Comment:
I would add a `--rm` so the container is cleaned up after it exits.. but again, doesn't really matter.